src/chart.zoom.js

```diff
@@ -159,7 +172,7 @@ function doPan(chartInstance, deltaX, deltaY) {
 
 		helpers.each(chartInstance.scales, function(scale, id) {
 			if (scale.isHorizontal() && directionEnabled(panMode, 'x') && deltaX !== 0) {
-				panScale(scale, deltaX);
+				panScale(scale, deltaX, helpers.getValueOrDefault(chartInstance.options, defaultOptions));
```

Comment:
You had already done it!

`function doPan(chartInstance, deltaX, deltaY) {`
`var panOptions = chartInstance.options.pan;`
`if (panOptions && helpers.getValueOrDefault(panOptions.enabled, defaultOptions.pan.enabled)) {`
		`var panMode = helpers.getValueOrDefault(chartInstance.options.pan.mode, defaultOptions.pan.mode);`